foundations/foundation-config/src/main/java/io/servicecomb/config/ConfigUtil.java

```diff
@@ -169,4 +201,45 @@ public static void installDynamicConfig() {
         AbstractConfiguration dynamicConfig = ConfigUtil.createDynamicConfig();
         ConfigurationManager.install(dynamicConfig);
     }
+
+  private static class ServiceCombPropertyUpdateListener implements WatchedUpdateListener {
+
+    private final ConcurrentMapConfiguration injectConfig;
+
+    ServiceCombPropertyUpdateListener(ConcurrentMapConfiguration injectConfig) {
+      this.injectConfig = injectConfig;
+    }
+
+    @Override
+    public void updateConfiguration(WatchedUpdateResult watchedUpdateResult) {
```

Comment:
i'm not sure, if "cse" will part of the config key not just the prefix, like: abc.cse.123 or abc.csexxx.123
maybe just change "cse." prefix to "servicecomb." is better?